.github/workflows/deploy.yml

```diff
@@ -42,6 +42,7 @@ jobs:
         uses: JamesIves/github-pages-deploy-action@3.7.1
         with:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          OPENCOLLECTIVE_API_KEY: ${{ secrets.OPENCOLLECTIVE_API_KEY }}
```

Comment:
Which account holds this key?

Reply:
No longer used, could be removed. PR here https://github.com/webpack/webpack.js.org/pull/4390